tests/system/src/test/java/test/robot/javafx/scene/NodeInitializationStressTest.java

```diff
@@ -0,0 +1,947 @@
+            accessControl(c);
+        });
+    }
+
+    @Test
+    public void text() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            return new Text(nextString());
+        }, (c) -> {
+            c.setText(nextString());
+            accessNode(c);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void textArea() {
+        test(() -> {
+            TextArea c = new TextArea();
+            c.setSkin(new TextAreaSkin(c));
+            return c;
+        }, (c) -> {
+            accessTextInputControl(c);
+        });
+    }
+
+    @Test
+    public void textField() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            TextField c = new TextField();
+            c.setSkin(new TextFieldSkin(c));
+            return c;
+        }, (c) -> {
+            accessTextInputControl(c);
+            c.setAlignment(Pos.CENTER);
+            c.getCharacters();
+        });
+    }
+
+    @Test
+    public void textFlow() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            TextFlow c = new TextFlow();
+            c.getChildren().setAll(createTextItems());
+            return c;
+        }, (c) -> {
+            c.getChildren().setAll(createTextItems());
+            accessRegion(c);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void titledPane() {
+        test(() -> {
+            TitledPane c = new TitledPane("TitledPane", null);
+            c.setSkin(new TitledPaneSkin(c));
+            return c;
+        }, (c) -> {
+            c.setAnimated(nextBoolean());
+            c.setExpanded(nextBoolean());
+            c.setContent(new Label(nextString()));
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8347392") // FIX
+    @Test
+    public void toggleButton() {
+        test(() -> {
+            ToggleButton c = new ToggleButton("ToggleButton");
+            c.setSkin(new ToggleButtonSkin(c));
+            return c;
+        }, (c) -> {
+            accessControl(c);
+            c.setSelected(nextBoolean());
+        });
+    }
+
+    @Test
+    public void toolBar() {
+        assumeFalse(SKIP_TEST);
+        test(() -> {
+            ToolBar c = new ToolBar();
+            c.setSkin(new ToolBarSkin(c));
+            c.getItems().setAll(createButtons());
+            return c;
+        }, (c) -> {
+            c.getItems().setAll(createButtons());
+            accessControl(c);
+        });
+    }
+
+    @Disabled("JDK-8348100") // FIX
+    @Test
+    public void tooltip() {
+        test(() -> {
+            Tooltip t = new Tooltip("this is a tooltip");
+            t.setShowDelay(Duration.ZERO);
+            t.setHideDelay(Duration.ZERO);
+            Label c = new Label("testing tooltip");
+            c.setSkin(new LabelSkin(c));
+            c.setTooltip(t);
+            c.setId("Tooltip");
+            return c;
+        }, (c) -> {
+            Tooltip t = c.getTooltip();
```

Comment:
For the background thread, construct a new object.

Reply:
I am going to address the tooltip issue in a different PR so as not to prolong the review process...